app/domain/repositories/UserRepository.scala

```diff
@@ -1,11 +1,22 @@
 package domain.repositories
 
+import domain.entity.User
+
 class UserRepository {
-  def login(username: String, password: String): String ={
-    username
+  def login(username: String, password: String): User ={
+    //TODO: change this to get actual user
```

Comment:
@Bukarachka in case of TODOs please create a ticket for each TODO and add a link to it here. 
So for this one, it will be the following:

//TODO: change this to get actual user https://trello.com/c/YH0aogAI/86-create-db-entities